client/verta/tests/conftest.py

```diff
@@ -356,14 +356,17 @@ def commit(repository):
 
 
 @pytest.fixture
-def created_datasets(client):
+def created_datasets(client, client_2):
```

Comment:
We can't add `client_2` as a requisite for this fixture because `client_2` causes the test to skip if its auth creds aren't present, so this would skip tests that it shouldn't.

Fortunately, I do believe `dataset`s have their own `_conn` that should be usable for the deletion (see below).
```suggestion
def created_datasets():
```

Reply:
Fixed.